### PR TITLE
[luci-micro] Copy PAL from luci-interpreter to luci-micro

### PR DESCRIPTION
**Pull request by SlavikMIPT**
This commit copies PAL from luci-interpreter to luci-micro.

ONE-DCO-1.0-Signed-off-by: Vyacheslav Bazhenov <v.bazhenov@partner.samsung.com>

### CODE REVIEW

**Review by BalyshevArtem — APPROVED** [review]
LGTM

**Review by binarman — APPROVED** [review]
LGTM

**Comment by binarman** [comment]
To think: remove these PALs from original luci-interpreter.

File: compiler/luci-micro/CMakeLists.txt [hunk 1]
```diff
@@ -19,7 +19,7 @@ set(CMAKE_ARM_OPTIONS
   "-DEXT_OVERLAY_DIR=${CMAKE_CURRENT_BINARY_DIR}/../../overlay"
   "-DFlatbuffers_DIR=${CMAKE_CURRENT_BINARY_DIR}/../../overlay/lib/cmake/flatbuffers"
   "-DCMAKE_TOOLCHAIN_FILE=${NNAS_PROJECT_SOURCE_DIR}/infra/nncc/cmake/buildtool/config/arm-none-eabi-gcc.cmake"
-  "-DLUCI_INTERPRETER_PAL_DIR=${CMAKE_CURRENT_SOURCE_DIR}/../luci-interpreter/pal/mcu"
+  "-DLUCI_INTERPRETER_PAL_DIR=${CMAKE_CURRENT_SOURCE_DIR}/luci-interpreter/pal/mcu"
```
Comment (binarman) [inline compiler/luci-micro/CMakeLists.txt:22]: Note: need to make it possible to set different PAl while project configuration without change of this CMakeLists

**Pull request merged**